src/common/onInit.ts

```diff
@@ -0,0 +1,39 @@
+import * as logger from "../logger";
+
+let initCallback: (() => unknown) | null = null;
+let didInit = false;
```

Comment:
Do we leave globals as camel-cased or in CAPS?

Reply:
Typically CAPS is for consts, which these aren't